From 2a441f04864b8c69d3f2d1b777dd943b49eb929b Mon Sep 17 00:00:00 2001
From: Josh Poimboeuf <jpoimboe@redhat.com>
Date: Thu, 4 Sep 2014 18:54:35 -0400
Subject: [kernel] ftrace-graph: Remove usage of ftrace_stop() in ftrace_graph_stop()

Message-id: <c40116de1535ddc13d78602f535377bf11b849a0.1409856554.git.jpoimboe@redhat.com>
Patchwork-id: 91247
O-Subject: [RHEL7.1 PATCH 16/23] ftrace-graph: Remove usage of ftrace_stop() in ftrace_graph_stop()
Bugzilla: 1113829
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1113829

commit 545d47b8f359f7e9b5beabc28bfeecb3fc6af1ee
Author: Steven Rostedt (Red Hat) <rostedt@goodmis.org>
Date:   Wed Jun 25 11:13:27 2014 -0400

    ftrace-graph: Remove usage of ftrace_stop() in ftrace_graph_stop()

    All archs now use ftrace_graph_is_dead() to stop function graph
    tracing. Remove the usage of ftrace_stop() as that is no longer
    needed.

    Cc: Frederic Weisbecker <fweisbec@gmail.com>
    Reviewed-by: Masami Hiramatsu <masami.hiramatsu.pt@hitachi.com>
    Signed-off-by: Steven Rostedt <rostedt@goodmis.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/kernel/trace/trace_functions_graph.c b/kernel/trace/trace_functions_graph.c
index c462c5b..4214e82 100644
--- a/kernel/trace/trace_functions_graph.c
+++ b/kernel/trace/trace_functions_graph.c
@@ -40,11 +40,6 @@ bool ftrace_graph_is_dead(void)
 void ftrace_graph_stop(void)
 {
  kill_ftrace_graph = true;
- /*
-  * ftrace_stop() will be removed when all archs are updated to
-  * use ftrace_graph_is_dead()
-  */
- ftrace_stop();
 }
 
 /* When set, irq functions will be ignored */
-- 
1.7.1